diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/FileStructureRule.kt

```diff
@@ -75,6 +75,11 @@ class FileStructureRule(configRules: List<RulesConfig>) : DiktatRule(
     private var packageName = ""
     private val ignoreImports = setOf("invoke", "get", "set")
 
+    // Fixme: this imports should be deleted if unused
+    private val ignoreOperators = setOf("getValue", "plus", "minus", "times", "div",
+        "rem", "rangeTo", "contains", "plusAssign", "minusAssign", "timesAssign",
+        "divAssign", "remAssign", "compareTo", "inc", "dec", "unaryPlus", "unaryMinus", "not")
+
```

Comment:
Can we supplement the dictionary with _operatorMap_ operators in _AstConstants.kt_ so that these operators are not ignored?

Reply:
Fixed